### PR TITLE
RELAP5-3D plugin

### PR DESCRIPTION
**Pull request by zhieejhia93**
A plugin to execute RELAP5-3D with WATTS. Several things to highlight:

- The default `run_proc()` does not work with RELAP5. Seems like it's because of the extra argument of 'stdout' to subprocess.Popen(). As a work around, I explicitly use subprocess.Popen() in the plugin to run the executable.
- This version of RELAP5-3D does not generate a CSV file. Instead, it generates a text file with a specific formatting. The text file needs to be converted to csv before the results can be extracted by the plugin. Other (newer) versions of RELAP5-3D is able to generate output csv files directly. The text-to-csv conversion in the plugin may need to be updated or removed in the future. 
- I also updated the documentation, added an example, and updated the test case for the new plugin.

### CODE REVIEW

File: doc/source/user/usage.rst [hunk 1]
```diff
@@ -251,14 +251,57 @@ Similar to the SAS executable, the utilities are also OS dependent. To execute
 SAS, the :meth:`~watts.PluginSAS` instance is called directly in the same way as
 other plugins.
 
-WATTS also supports the "INCLUDE" directive of SAS that allows for multiple input
-files. Users simply need to add "INCLUDE extra_input_file_names" to the template
-and specify the names of the extra input files as a string to the
-"extra_template_inputs" argument when calling the :class:`~watts.PluginSAS` class::
+SAS4A/SASSY-1 Plugin
+~~~~~~~~~~~~~~~~~~~~
```
Comment (nstauff) [inline doc/source/user/usage.rst:269]: wrong title.

File: examples/1App_RELAP5_SingleChannel/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## Keywords
+ 
+- Single annulus channel
+- Internally heated
+
+## File descriptions
+
+- [__watts_exec.py__](watts_exec.py): This is the file to execute to run the problem described above.
+- [__relap5_template__](relap5_template): Templated SAS input.
```
Comment (nstauff) [inline examples/1App_RELAP5_SingleChannel/README.md]: update to `Templated RELAP5 input.`

**Review by nstauff — CHANGES_REQUESTED** [review]
thanks for the plugin - I just had minor editorial revisions.

**Comment by nstauff** [comment]
@lingzou I am adding you as reviewer - could you please test this plugin out? I don't have access to RELAP5 so would not be able to check it works. Thanks!

**Comment by lingzou** [comment]
> @lingzou I am adding you as reviewer - could you please test this plugin out? I don't have access to RELAP5 so would not be able to check it works. Thanks!

@nstauff If not urgent, I will test it a bit later after setting up RELAP5 on my computer.

**Comment by nstauff** [comment]
> > @lingzou I am adding you as reviewer - could you please test this plugin out? I don't have access to RELAP5 so would not be able to check it works. Thanks!
> 
> @nstauff If not urgent, I will test it a bit later after setting up RELAP5 on my computer.

Yes, this is not urgent. thanks!

**Comment by lingzou** [comment]
@nstauff As discussed, I only performed a "visual" check on the MR, and it looks good to me.

**Comment by lingzou** [comment]
@nstauff As discussed, I only performed a "visual" check on the MR, and it looks good to me.

**Comment by nstauff** [comment]
Thanks @lingzou - I will approve the PR once @zhieejhia93 addresses my minor editorial revisions requested!

**Review by nstauff — APPROVED** [review]

**Comment by nstauff** [comment]
Thanks @zhieejhia93 - note that the CI tests are failing. Could you check what is going on?

**Comment by zhieejhia93** [comment]
@nstauff Ya, I am looking into that right now. Seems to be due to the changes that Paul made recently. I should be able to fix that.

**Comment by zhieejhia93** [comment]
@nstauff The changes passed the CI tests. Everything should be good now. Thanks

**Pull request merged**